crates/bevy_render/src/camera/camera.rs

```diff
@@ -585,6 +585,8 @@ pub fn camera_system<T: CameraProjection + Component>(
         .filter_map(|event| {
             if let AssetEvent::Modified { id } = event {
                 Some(id)
+            } else if let AssetEvent::Added { id } = event {
+                Some(id)
             } else {
                 None
             }
```

Comment:
```suggestion
            AssetEvent::Modified { id } | AssetEvent::Added { id } => Some(id),
            _ => None,
```
Since we are adding another check on `AssetEvent`, a `match` fits better.

Reply:
Ah yes, nice catch.